### ISSUE TITLE
Services no longer support "export"

### ISSUE THREAD

**Issue by ethernetdan** [kubernetes/kubernetes#22247]
It appears that ability to export services no longer works in recent builds of Kubernetes. This problem surfaced in redspread/spread#67.

The error is produced in [/pkg/apiserver/resthandler.go:137](https://github.com/kubernetes/kubernetes/blob/a608cc9/pkg/apiserver/resthandler.go#L137) 

### kubectl
```
$ kubectl get --export=true services/example-service

Error from server: export of "services" is not supported
```

### API
`curl -v "localhost:9000/api/v1/namespaces/default/services/example-service?export=1"`
##### Request
```
GET /api/v1/namespaces/default/services/example-service?export=1 HTTP/1.1
Host: localhost:9000
User-Agent: curl/7.43.0
Accept: */*
```
##### Response
```
HTTP/1.1 400 Bad Request
Content-Type: application/json
Date: Tue, 01 Mar 2016 02:56:18 GMT
Content-Length: 182

{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "export of \"services\" is not supported",
  "reason": "BadRequest",
  "code": 400
}
```

**Comment by AdoHe** [comment]
I test this under `1.2.0-alpha8`, and meet the same problem.
@bgrant0607 I am afraid we should still support service export in latest builds. If this should be supported, I would like to take a look.

**Comment by bgrant0607** [comment]
Yes, this is a bug

**Comment by bgrant0607** [comment]
cc @kubernetes/sig-api-machinery

**Comment by AdoHe** [comment]
I would take a look today.

**Comment by bgrant0607** [comment]
Thanks @AdoHe.

It looks like the last thing that touched this was https://github.com/smarterclayton/kubernetes/commit/125ef6fbc8cc70477071936cd93d064d71c773e6

cc @smarterclayton

**Comment by AdoHe** [comment]
The root cause is because this struct [REST](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/service/rest.go#L49) does not implement [Exporter](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/rest/rest.go#L76) interface. I have fixed this and will open a PR asap.

**Comment by AdoHe** [comment]
Opened a PR to fix this #22285

**Comment by ethernetdan** [comment]
Confirmed resolved.
